model/src/main/kotlin/LicenseFinding.kt

```diff
@@ -54,8 +54,34 @@ data class LicenseFinding @JsonCreator constructor(
         val license: String,
         val copyrights: SortedSet<String>
 ) : Comparable<LicenseFinding> {
+    companion object {
+        private val copyrightsComparator = Comparator<SortedSet<String>> { o1, o2 ->
```

Comment:
In our case `copyrights` is always a `TreeSet`, so I wonder whether we should simply cast to a `TreeSet` and use its `comparator()` (provided it returns non-null).

Reply:
This is not about comparing the elements of the sets, but about comparing the sets, so this comparator would not help. Also, it is always null in this case because we never set it. I am moving this to a separate class in another commit which will be in the next PR, I will use the `SortedSet.comparator()` there in case it is not null for comparing the elements.

Reply:
AFAIK there is no `SortedSet.comparator()` as `SortedSet` is an interface...

Reply:
Yes there is: https://docs.oracle.com/javase/8/docs/api/java/util/SortedSet.html#comparator--
Anyway, I decided not to use it in the other commit because both sets could have different comparators, but that can be discussed in the other PR then :)